core/python_utils.py

```diff
@@ -384,32 +384,6 @@ def url_encode(query, doseq=False):
     return urlparse.urlencode(query, doseq=doseq)
 
 
-def url_retrieve(source_url, filename=None):
```

Comment:
Remove `url_retrieve` from .pylintrc file and general_purpose_linter.py also.

Reply:
@Hudda  
Already Done

Reply:
Yep -- I missed that, but you still have not removed tests from general_purpose_linter.py